packages/webapp/public/locales/en/translation.json

```diff
@@ -1071,7 +1071,16 @@
       "YOU_WILL_FIND": "Here you will find:"
     }
   },
+  "ENTITY_TYPES": {
+    "TASK": "task",
+    "location": "location"
```

Comment:
Is there a reason this key name is in lower case? It seems inconsistent, possibly incorrect (I know sometimes strings are converted to upper case for uses like these).